action.yml

```diff
@@ -16,7 +16,7 @@ inputs:
   ape-plugins-list:
     description: 'Space seperated list of plugins to install with relevant pinning applied'
     required: False
-    default: '-U .'
+    default: ''
```

Comment:
Nope, I don't think so anyway.

This is how it works:

If the user is _not_ specifying anything here (using the default), it will calculate a default.
if we make the default ".", it would not be very easy to detect whether the user has specified "." or they are using the default behavior.